### ISSUE TITLE
Deerskin Gloves don't have first person model

### ISSUE THREAD

**Issue by WH1T3No1SE** [HAHAYOUDEAD/NorthfolkClothingPack#7]
In recent update Hinterland added first person gloves models, Deerskin Gloves do not have it.

**Comment by HAHAYOUDEAD** [comment]
obviously, that feature wasn't there when the mod was released. Will add those when Modcomponent has this feature too
